packages/samples/vuejs/README.md

```diff
@@ -1,19 +1,231 @@
+If you are using the npm package and seeing the warning bellow, you might want to switch to using the CDN, as there is currently [a small issue](https://github.com/ionic-team/stencil/issues/3195) with Stencil and Vite. This shouldn't affect the application however, and you can suppress the warning if you prefer.
+
+```txt
+The above dynamic import cannot be analyzed by vite.
+See https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations for supported dynamic import formats. If this is intended to be left as-is, you can use the /* @vite-ignore */ comment inside the import() call to suppress this warning.
 ```
-npm run serve
+
+Usage for both the npm package and CDN are explained in separate sections further in this file.
+
+## Configuration
+
+Add Vue configuration to help it handle atomic's components
+
+```js
+// vue.config.js
+{
+  ...,
+  chainWebpack: (config) => {
+    // lets us parse html templates, which will be necessary for atomic components that require the native <template> tag
+    config.module
+      .rule('html')
+      .test(/\.html$/)
+      .use('html-loader')
+      .loader('html-loader');
+
+    // tells Vue to treat any components starting with 'atomic-' as native components, and therefore not try and create Vue components out of them
+    config.module
+      .rule('vue')
+      .use('vue-loader')
+      .tap((options) => ({
+        ...options,
+        compilerOptions: {
+          isCustomElement: (tag) => tag.startsWith('atomic-'),
+        },
+      }));
+  }
+}
+```
+
+Add the `html-loader` dev-dependency
+
+```sh
+npm install --save-dev html-loader
+```
+
+## Using the `@coveo/atomic` npm package
+
+### Add dependencies
+
+Install `@coveo/atomic`
+
+```sh
+npm install --save @coveo/atomic
+```
+
+Install `ncp`
+
+```sh
+npm install --save-dev ncp
+```
+
+> `ncp` is a utility that we will use to copy assets from the atomic project into the vue application, as explained below.
+
+### Copy assets from `@coveo/atomic`
+
+In order for you to be able to leverage everything that atomic has to offer, you will need to copy some assets into your public folder. If you skip this step, some of your dynamic content that is based on localization or svgs will be missing, and you will see some content being displayed as the placeholder 'between-parentheses' instead.
+
+```json
+scripts: {
+  ...,
+  "dev": "npm run copy:assets && npm run serve",
+  "copy:assets": "ncp node_modules/@coveo/atomic/dist/atomic/assets public/assets && ncp node_modules/@coveo/atomic/dist/atomic/lang public/lang"
+}
+```
+
+> The `dev` script assumes you are using the default `serve` script and `public/assets` path for running your application. Don't forget to adjust if needed.
+
+### Update your entry file
+
+The atomic package allows you to customise your components by defining CSS variables in your own stylesheet - you can read about that [here](https://docs.coveo.com/en/atomic/latest/usage/themes-and-visual-customization/).
```

Comment:
```suggestion
The Atomic package allows you to customize your components by defining CSS variables in your own stylesheet (see [Themes and Visual Customization](https://docs.coveo.com/en/atomic/latest/usage/themes-and-visual-customization/)).
```
We generally use North American English 🙃

Reply:
sorry, I'm tryinggg

Reply:
No worries at all, as I wrote above your doc looks very good 😄 👍 
I just didn't want my comment to look arbitrary 😉